### PR TITLE
[ros2] lint the pcl_ros code base to try and get a green build :)

### PR DESCRIPTION
**Pull request by asymingt**
Mostly header ordering, but also some uncrustification and a few nitty pep8 errors.

### CODE REVIEW

**Comment by asymingt** [comment]
**GREEN GREEN GREEN**

happy

File: pcl_ros/include/pcl_ros/pcl_node.hpp [hunk 1]
```diff
@@ -249,7 +244,9 @@ class PCLNode : public rclcpp::Node
     * \param topic_name an optional topic name (only used for printing, defaults to "indices")
     */
   inline bool
-  isValid(const PointIndicesConstPtr & indices, const std::string & topic_name = "indices")
+  isValid(
+    [[maybe_unused]] const PointIndicesConstPtr & indices,
```
Comment (SteveMacenski) [inline pcl_ros/include/pcl_ros/pcl_node.hpp]: For just defining, you can not post a name or comment out via `/*indices*/`. This is typically how codebases I've seen handle this issue - but I didn't actually know about this `maybe_unused` option. This is what I'd prefer if possible, but I won't die on a hill about it

Comment (SteveMacenski) [reply pcl_ros/include/pcl_ros/pcl_node.hpp]: These appear to be the only instances, so its not a huge delta change (I probably wouldn't ask you to do it if it required changes 20x+)

**Review by SteveMacenski — CHANGES_REQUESTED** [review]
Thanks so much for this! This makes reviewing PRs much easier for me :-)

**Comment by SteveMacenski** [comment]
I also wish there was an easier gif-generator-poster bot so I could have a bunch of funny keywords result in memes :laughing:

File: pcl_ros/include/pcl_ros/pcl_node.hpp (hunk 1 above)
Comment (asymingt) [reply pcl_ros/include/pcl_ros/pcl_node.hpp]: Ok I replaced the four instances of `[[maybe_unused]]` with commenting out.

**Review by SteveMacenski — APPROVED** [review]

**Pull request merged**